site/en_AU.all.json

```diff
@@ -433,7 +461,7 @@
   "shopping_card_new_subscription": { "other": "This card will be saved and charged regularly" },
   "shopping_card_change": { "other": "Credit card not correct? <a href=\"{{.SubscriptionsURL}}\">Click here to update your card.</a>" },
 
-  "shopping_info_ownership_plan": { "other": "Subscription"},
+  "shopping_info_ownership_plan": { "other": "shopping_info_ownership_plan" },
```

Comment:
What's going on here?

Reply:
Hmmm, nice observation. Wonder if I did a merge of my local develop branch rather than remote

Reply:
I noticed a few translations like that